tools/install.sh

```diff
@@ -110,21 +129,109 @@ echo "Downloading the release: '${ANKAIOS_RELEASE_URL}'"
 download_release "${ANKAIOS_RELEASE_URL_SHA}"
 download_release "${ANKAIOS_RELEASE_URL}"
 
-# skip checksum validation if sha512sum is not available
+# Skip checksum validation if sha512sum is not available
 if command -v sha512sum >/dev/null; then
     echo "Checking file checksum"
     sha512sum -c "${RELEASE_FILE_NAME_WITH_SHA}"
 else
     echo "Warning: 'sha512sum' not installed. Skipping checksum validation."
 fi
 
-# prefix with sudo if install dir is not writeable with current permissions
-PREFIX="sudo"
+# Prefix with sudo if install dir is not writeable with current permissions
+BIN_SUDO="sudo"
 if [ -w "${BIN_DESTINATION}" ]; then
-    PREFIX=""
+    BIN_SUDO=""
 fi
 
 echo "Extracting the binaries into install folder: '${BIN_DESTINATION}'"
-${PREFIX} tar -xvzf "${RELEASE_FILE_NAME}" -C "${BIN_DESTINATION}"
+${BIN_SUDO} tar -xvzf "${RELEASE_FILE_NAME}" -C "${BIN_DESTINATION}/"
+
+# Install systemd unit files
+if [ -d "$SERVICE_DEST" ]; then
+    SVC_SUDO="sudo"
+    if [ -w "$SERVICE_DEST" ]; then
+        SVC_SUDO=""
+    fi
+
+    if [[ "$INSTALL_TYPE" == server || "$INSTALL_TYPE" == both ]]; then
+        $SVC_SUDO tee "$FILE_ANK_SERVER_SERVICE" >/dev/null << EOF
+[Unit]
+Description=Ankaios server
+
+[Service]
+ExecStart=/usr/local/bin/ank-server $SERVER_OPT
+
+[Install]
+WantedBy=default.target
+EOF
+    echo "Start server with 'systemctl start $ANK_SERVER_SERVICE'"
+    fi
+
+    if [[ "$INSTALL_TYPE" == agent || "$INSTALL_TYPE" == both ]]; then
+        $SVC_SUDO tee "$FILE_ANK_AGENT_SERVICE" >/dev/null << EOF
+[Unit]
+Description=Ankaios agent
+
+[Service]
+ExecStart=/usr/local/bin/ank-agent $AGENT_OPT
```

Comment:
OK. I will replace `usr/local/bin` by `${BIN_DESTINATION}`. I think preventing writing the unit files in case no root-privileges are available is not worth the effort.